includes/admin/settings/class-wc-settings-accounts.php

```diff
@@ -244,7 +244,7 @@ public function get_settings() {
 					'title'         => __( 'Enable tracking', 'woocommerce' ),
 					'desc'          => __( 'Allow usage of WooCommerce to be tracked', 'woocommerce' ),
 					/* Translators: %s URL to tracking info screen. */
-					'desc_tip'      => sprintf( esc_html__( 'If you would rather opt-out, and do not check this box, we will not know this store exists and we will not collect any usage data. %s.', 'woocommerce' ), $tracking_info_text ),
+					'desc_tip'      => sprintf( esc_html__( 'To opt-out, simply leave this box unchecked. We won’t know this store exists and won’t collect any usage data. %s.', 'woocommerce' ), $tracking_info_text ),
```

Comment:
As suggested in https://wp.me/p2JRYi-402 by @californiakat, this one also alleviates the confusion around who is "we".

```suggestion
					'desc_tip'      => sprintf( esc_html__( 'To opt out, leave this box unticked. Your store remains untracked, and no data will be collected. %s.', 'woocommerce' ), $tracking_info_text ),
```